src/components/navigation/index.js

```diff
@@ -16,14 +16,12 @@ class Navigation extends Component {
       href="/login"
       children={'Log In'}
       key="login"
-      callBack={this.closeMenu}
```

Comment:
This removes closing of the menu when a link is clicked. I think this is not the solution that we want. We still want to pass this property to the Link component but we don't want it added to a DOM element.

If we look at the link component, the props are probably being spread into a DOM element, we probably still want to spread the props but delete any that shouldn't get passed to the DOM before spreading.